### ISSUE TITLE
Cider is useless

### ISSUE THREAD

**Issue by Austin5003** [al132mc/techemistry#7]
There's no further distillation product, and it can't even be put in the distillery controller

**Comment by ghost** [comment]
This is true, the mod is definitely still unfinished so this is something that I should add soon.
